public/app/features/explore/LogsMetaRow.tsx

```diff
@@ -78,7 +78,6 @@ export const LogsMetaRow: React.FC<Props> = React.memo(
           >
             <Button variant="secondary" size="sm" onClick={onEscapeNewlines}>
               <span>{forceEscape ? 'Remove escaping' : 'Escape newlines'}&nbsp;</span>
```

Comment:
maybe we don't need the whitespace if we removed the icon?

Reply:
I removed it.